.github/workflows/main.yml

```diff
@@ -35,10 +35,10 @@ jobs:
       - uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
         with:
           go-version: ${{ env.GOVERSION }}
-      - uses: arduino/setup-protoc@64c0c85d18e984422218383b81c52f8b077404d3 # v1.1.2
+      - uses: trail-of-forks/setup-protoc@a97892a429d98fae78d26f40334ab7eb616d08b9
```

Comment:
N.B. that this is still using my fork of the `setup-protoc` action, since they haven't merged my fixes yet.

Reply:
I'm good to use your fork until they do, thanks for getting this working!